.github/workflows/cmake.yml

```
@@ -116,6 +116,17 @@ jobs:
       run: cmake --build build --config ${{ env.cmake_build_type }} --parallel 2
     - name: install
       run: cmake --install build --config ${{ env.cmake_build_type }}
+    - name: "test configure"
```

```suggestion
    - name: "configure build-test"
```

@jcelerier I am going to rename these CI steps, as much as anything to force re-run CI to get the latest error logs (the old ones have expired). You'll have to pull the changes to your local working copy.